scripts/set-iscsi-initiator

```diff
@@ -46,8 +57,8 @@ TMPFILE=$TMPDIR/initiatorname.iscsi
                 exit 1
         fi
 
-        echo "InitiatorName=$1" > $TMPFILE
-        echo "InitiatorAlias=$2" >> $TMPFILE
+        echo "InitiatorName=$INITIATORNAME" > $TMPFILE
+        echo "InitiatorAlias=$INITIATORALIAS" >> $TMPFILE
 
         diff $INITIATORFILE $TMPFILE > /dev/null 2>&1 
```

Comment:
good point. The new bit is required to avoid getting hit with the 
```
there are active sessions so not updating
```
message if we're not changing anything, which happens when HA is activated